src/libraries/System.Runtime/tests/System.Runtime.Extensions.Tests/System/Environment.StackTrace.cs

```diff
@@ -23,7 +23,11 @@ public void StackTraceTest()
             {
                 "System.Tests.EnvironmentStackTrace.StaticFrame(Object obj)",
                 "System.Tests.EnvironmentStackTrace.TestClass..ctor()",
-                "System.Tests.EnvironmentStackTrace.GenericFrame[T1,T2](T1 t1, T2 t2)",
+#if DEBUG
```

Comment:
This test will likely fail in native aot outerloop testing.

Implementing this level of detail on native aot side is not possible without a size regression because we simply don't have this information in the file format. It is not an easy fix on native aot side.

Reply:
I expect such information only being emitted for methods with optimization disabled. So there shouldn't be any size regression for production.